Makefile

```diff
@@ -205,8 +205,8 @@ install-resources:
 
 .PHONY: e2e-test
 e2e-test: e2e-dependencies
-	$(GINKGO) -v --fail-fast $(E2E_TEST_ARGS) test/e2e
-
+	$(GINKGO) -v -p --fail-fast -procs=7 $(E2E_TEST_ARGS) test/e2e
+# 
```

Comment:
cuz our CI default is 2

Reply:
i wonder limitation looks more procs faster also I didn't request a review because I am not finished yet so weird.. I cannot even remove you from the reviewer list. you are the default reviewer now

Reply:
> I wonder limitation looks more procs faster also I didn't request a review because I am not finished yet so weird.. I cannot even remove you from the reviewer list. you are the default reviewer now

FYI for next time, if you open a Draft PR, it doesn't get assigned reviewers but I think workflows still run. 🙂

Reply:
oh!! Draft PR I will ask you next time Thanks for letting know good info!!! @dhaiducek